packages/block-directory/src/components/downloadable-block-author-info/style.scss

```diff
@@ -0,0 +1,9 @@
+.block-directory-downloadable-block-author-info__content {
```

Comment:
There's now a new 'tip' on the right hand side of the inserter, and I think it'd be good to make the line height of the block description match that:
<img width="693" alt="Screen Shot 2019-09-13 at 12 39 00 pm" src="https://user-images.githubusercontent.com/677833/64879391-e1ae3280-d623-11e9-96f3-2a1ac9b7079e.png">